content/geometry/human-index.adoc

```diff
@@ -0,0 +1,30 @@
+= Human structure
+:toc:
+:toclevels: 7
+
+//include::human-structure/Floor.adoc[leveloffset=+1] Removed as stated on the Miro board
+include::human-structure/Grp_Equipment_Static_(T).adoc[leveloffset=+1]
+include::human-structure/Skeleton.adoc[leveloffset=+1]
+include::human-structure/Hip_(T).adoc[leveloffset=+2]
+include::human-structure/Lower_Spine_(T).adoc[leveloffset=+3]
+include::human-structure/Upper_Spine_(T).adoc[leveloffset=+4]
+include::human-structure/Neck_(T).adoc[leveloffset=+5]
+include::human-structure/Head_(T).adoc[leveloffset=+6]
+include::human-structure/Shoulder_R_(T).adoc[leveloffset=+4]
+include::human-structure/Upper_Arm_R_(T).adoc[leveloffset=+5]
+include::human-structure/Lower_Arm_R_(T).adoc[leveloffset=+6]
+include::human-structure/Full_Hand_R_(T).adoc[leveloffset=+7]
+include::human-structure/Shoulder_L_(T).adoc[leveloffset=+4]
+include::human-structure/Upper_Arm_L_(T).adoc[leveloffset=+5]
+include::human-structure/Lower_Arm_L_(T).adoc[leveloffset=+6]
+include::human-structure/Full_Hand_L_(T).adoc[leveloffset=+7]
+include::human-structure/Upper_Leg_R_(T).adoc[leveloffset=+3]
+include::human-structure/Lower_Leg_R_(T).adoc[leveloffset=+4]
+include::human-structure/Full_Foot_R_(T).adoc[leveloffset=+5]
+include::human-structure/Upper_Leg_L_(T).adoc[leveloffset=+3]
+include::human-structure/Lower_Leg_L_(T).adoc[leveloffset=+4]
+include::human-structure/Full_Foot_L_(T).adoc[leveloffset=+5]
```

Comment:
In parallel to Full_Foot_X(T), as proposed on the Miro Board, we should implement a helper bone "Standing_Point" e.g.
|- Standing_Point_Right (T)